Zend/zend_enum.c

```diff
@@ -183,6 +185,63 @@ void zend_enum_add_interfaces(zend_class_entry *ce)
 	}
 }
 
+void zend_enum_build_backed_enum_table(zend_class_entry *ce)
+{
+	uint32_t backing_type = ce->enum_backing_type;
+	ZEND_ASSERT(backing_type != IS_UNDEF);
+
+	if (zend_update_class_constants(ce) == FAILURE) {
```

Comment:
An unfortunate consequence of this is that all enum values will be instantiated when the enum class is loaded :/

Reply:
Yeah... There might be a more appropriate place. Maybe even `zend_update_class_constants` itself?

Reply:
Although that also means the backing value validation wouldn't happen until the enum is used in some way. I'm not sure which one is preferable.